runtime/lua/vim/treesitter/highlighter.lua

```diff
@@ -75,7 +75,7 @@ local TSHighlighter = {
 
 TSHighlighter.__index = TSHighlighter
 
----@package
+---@nodoc
```

Comment:
what do we think about `@internal` instead of `@nodoc`? Seems a bit more flexible while still giving us the semantics we want.

Reply:
Good idea.